openstack/ces/v1/alarms/requests.go

```diff
@@ -0,0 +1,120 @@
+package alarms
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+)
+
+type ListAlarmsRequest struct {
+	// The value ranges from 1 to 100, and is 100 by default.
+	// This parameter is used to limit the number of query results.
+	Limit *int `q:"limit,omitempty"`
+	// Specifies the result sorting method, which is sorted by timestamp.
+	// The default value is desc.
+	// asc: The query results are displayed in the ascending order.
+	// desc: The query results are displayed in the descending order.
+	Order string `q:"order,omitempty"`
+	// Specifies the first queried alarm to be displayed on a page.
+	Start string `q:"start,omitempty"`
+}
+
+func ListAlarms(client *golangsdk.ServiceClient, req ListAlarmsRequest) (r ListAlarmsResult) {
```

Comment:
Opts argument should be a `OptsBuilder` interface, as was discussed in https://github.com/opentelekomcloud/gophertelekomcloud/issues/274